File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        count = 0
        seats.sort()
        students.sort()
        
        for i in range(len(students)):
            count += abs(students[i] - seats[i])
        
        return count

File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function(seats, students) {
    seats.sort((a,b) => a - b);
    students.sort((a,b) => a - b);
    
    //console.log(seats,students)
    let count = 0;
    let i = 0;
    while(i < students.length){
        count += Math.abs(students[i] - seats[i]);
        i++
    }
    return count
};